front_end/src/app/(main)/components/bulletin.tsx

```diff
@@ -13,14 +13,14 @@ const Bulletin: FC<{ text: string; id: number }> = ({ text, id }) => {
   return (
     <div
       className={classNames(
-        "relative mb-2 mt-2 flex flex-col items-start rounded border-2 border-solid border-blue-700 bg-blue-400 p-4 pr-8 pt-6 dark:border-blue-700 dark:bg-blue-800",
+        "relative flex flex-col items-start rounded border border-solid border-blue-700/50 bg-blue-400/75 p-4 pr-8 text-blue-800 dark:border-blue-700/75 dark:bg-blue-800 dark:text-blue-200",
```

Comment:
Could we use the same color level for light and dark themes? For example, 

`text-blue-800 dark:text-blue-800-dark`

`text-blue-800` is close to `dark:text-blue-200`but not exactly, and will vary depending on the color pallette. And this pattern will inhibit our future ability to adjust the light/dark pallettes separately from each other.

Reply:
That raises a topic that I've defended for a long time but could not convince engineering - I'll fix this for now and explain my thinking in a larger colors proposal